lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -469,6 +475,7 @@ def __init__(self, client, results):
         self.load_path = parameters.get('load_path')
         self.name = parameters.get('name')
         self.network = build.get('network')
+        self.extra_hosts = build.get('etc_hosts')
```

Comment:
You should pass the `dict` through [clean_dict_booleans_for_docker_api](https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/docker/common.py#L902), or manually make sure that it only contains strings.